auto_post_classifier/main.py

```diff
@@ -5,16 +5,23 @@
 import pandas as pd
 import typer
 from typing_extensions import Annotated
+from typing import Optional
+
 from loguru import logger
 from rich.console import Console
 from pathlib import Path
 from auto_post_classifier import utils
 from auto_post_classifier.models import TaskBase
+from dotenv import load_dotenv
+
 import openai
+import uvicorn
+
 
 # todo: change the Path object
+load_dotenv()
```

Comment:
Having this line, I assume you're using a `.env` file which you rightfully don't track using git. In this case, I suggest creating `example.env` that *will* be tracked so other user/devs could quickly pick up.